sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -150,6 +163,39 @@ export class Sender {
     sendBatch(messages: SendableMessageInfo[]): Promise<void>;
     }
 
+// @public
+export class ServiceBusAtomManagementClient extends ServiceClient {
+    constructor(connectionString: any, options?: ServiceClientOptions);
+    createQueue(queuePath: string, options: any): Promise<HttpOperationResponse>;
+    createRule(topicPath: string, subscriptionPath: string, rule: string, options: any): Promise<HttpOperationResponse>;
+    createSubscription(topicPath: string, subscriptionPath: string, options: any): Promise<HttpOperationResponse>;
+    createTopic(topicPath: string, options: any): Promise<HttpOperationResponse>;
+    deleteQueue(queuePath: any): any;
+    deleteRule(topicPath: string, subscriptionPath: string, rule: string): any;
+    deleteSubscription(topicPath: string, subscriptionPath: string): any;
+    deleteTopic(topicPath: any): any;
+    // (undocumented)
+    endpoint: any;
+    formatDeadLetterPath(queuePath: any): any;
+    getQueue(queuePath: string): Promise<HttpOperationResponse>;
+    getRule(topicPath: string, subscriptionPath: string, rule: string): Promise<HttpOperationResponse>;
+    getSubscription(topicPath: string, subscriptionPath: string): Promise<HttpOperationResponse>;
+    getTopic(topicPath: string): Promise<HttpOperationResponse>;
+    listQueues(options?: any): Promise<HttpOperationResponse>;
+    listRules(topicPath: string, subscriptionPath: string, options?: any): Promise<HttpOperationResponse>;
+    listSubscriptions(topicPath: string, options?: any): Promise<HttpOperationResponse>;
+    listTopics(options?: any): Promise<HttpOperationResponse>;
+    // (undocumented)
+    queueResourceSerializer: ResourceSerializer;
+    // (undocumented)
+    ruleResourceSerializer: ResourceSerializer;
+    // (undocumented)
+    subscriptionResourceSerializer: ResourceSerializer;
+    // (undocumented)
+    topicResourceSerializer: ResourceSerializer;
+    updateQueue(queuePath: any, options: any): Promise<HttpOperationResponse>;
```

Comment:
updateRule, updateTopic and updateSubscription are missing

Reply:
Re-generated the file - this can be looked at in end before merging.

Addressed all comments so far, couple of points 
- Added in tests that take `azure-sb` as dev dependency to ensure correctness of structure and output is achieved. Once we are ready to merge, this can be removed.
- There are some /inconsistencies  in the way error info is populated in response for the get API. This would be something the service would need to fix.